build/installer.nsh

```diff
@@ -4,22 +4,44 @@
 ; Adding custom installation steps for electron-builder, ref:
 ; https://github.com/electron-userland/electron-builder/wiki/NSIS
 !macro customInstall
+  ; ===============================================================
+  ; Installation of drivers for dfu trigger and cdc acm
+  ; ===============================================================
 
-  ; The version of the bundled nRF5x-Command-Line-Tools
-  Var /GLOBAL BUNDLED_NRFJPROG_VERSION
-  StrCpy $BUNDLED_NRFJPROG_VERSION "9.7.1"
+  ; Install drivers for DFU trigger and CDC ACM
+  ; Put the files to the output directory.
+  File "${BUILD_RESOURCES_DIR}\drivers\driver-installer.exe"
+
+  ClearErrors
```

Comment:
Do the `ClearErrors` statements have any effect?

Reply:
They have no effect anymore. Will remove them.